File: dehaze-go/internal/model/sys_member_sign_in.go
package model

import "time"

type SysMemberSignIn struct {
	ID             int64     `gorm:"primaryKey;autoIncrement;column:id" json:"id"`
	UserID         int64     `gorm:"column:user_id;type:bigint;not null;uniqueIndex:uk_user_sign_date,priority:1;comment:用户ID" json:"userId"`
	SignDate       time.Time `gorm:"column:sign_date;type:date;not null;uniqueIndex:uk_user_sign_date,priority:2;index:idx_sign_date;comment:签到日期" json:"signDate"`
	ContinuousDays int       `gorm:"column:continuous_days;type:int;not null;default:1;comment:连续签到天数" json:"continuousDays"`
	GrowthValue    int       `gorm:"column:growth_value;type:int;not null;default:0;comment:本次获得成长值" json:"growthValue"`
	CreateTime     time.Time `gorm:"column:create_time;type:datetime;not null;default:CURRENT_TIMESTAMP;comment:创建时间" json:"createTime"`
}

func (SysMemberSignIn) TableName() string {
	return "sys_member_sign_in"
}

File: dehaze-python/app/models/entity/sys_member_sign_in.py
from datetime import date

from sqlalchemy import BigInteger, Date, Integer
from sqlalchemy.orm import Mapped, mapped_column

from app.models.base import AppendOnlyModel


class SysMemberSignIn(AppendOnlyModel):
    """会员签到记录表（只追加，不使用逻辑删除）。"""

    __tablename__ = "sys_member_sign_in"
    __table_args__ = {"comment": "会员签到记录表"}

    id: Mapped[int] = mapped_column(
        BigInteger, primary_key=True, autoincrement=True, comment="主键"
    )
    user_id: Mapped[int] = mapped_column(BigInteger, nullable=False, comment="用户ID")
    sign_date: Mapped[date] = mapped_column(Date, nullable=False, comment="签到日期")
    continuous_days: Mapped[int] = mapped_column(
        Integer, nullable=False, default=1, comment="连续签到天数"
    )
    growth_value: Mapped[int] = mapped_column(
        Integer, nullable=False, default=0, comment="本次获得成长值"
    )
